time stamp
int32 id  # ID must be echoed back in SampleResult
int32 arm  # which arm to relax (ActuatorType enum)
